features/support/pages/element_collection.rb

```diff
@@ -0,0 +1,21 @@
+# Convenience methods for working with many elements that represent objects
+class ElementCollection < SimpleDelegator
+  def element_for(object)
+    find do |element|
+      if element.respond_to?(:id)
```

Comment:
May be worth extracting this to a switch statement that is more intent revealing.

Reply:
Probably a good instinct long-term, but I wouldn't worry about it for now